Float.FileDownloader.Tests/DownloadRequest.Tests.cs

```diff
@@ -72,5 +74,18 @@ public async Task TestServerNotFound()
             var request = new HttpRequestMessage(HttpMethod.Get, "http://www.floatfloatfloat.float/float");
             await Assert.ThrowsAsync<HttpRequestException>(async () => await DownloadRequest.Download(request, TempFilePath()));
         }
+
+        [Fact]
+        public async Task TestNoCookies()
+        {
+            var request1 = new HttpRequestMessage(HttpMethod.Put, TestUriString());
+            await DownloadRequest.Download(request1, TempFilePath());
+
+            var request2 = new HttpRequestMessage(HttpMethod.Get, TestUri());
+            await DownloadRequest.Download(request2, TempFilePath());
+
+            var requestResult = await DownloadRequest.Download(request2, TempFilePath());
+            Assert.Empty(requestResult.Headers.Where(header => header.Key == "Set-Cookie"));
```

Comment:
@EBusch I don't think this is testing the right thing -- this is testing whether the server is returning a `Set-Cookie` response header (which we can't control). What we want to avoid is a request including the `Cookie` header.

Reply:
@mediabounds Turns out I wasn't testing the way I thought I was... I wrote the test, it went red, I changed it up and it went green... It went red because there was an error where the cookie wasn't adding properly. 🙃 Perhaps the best thing to do here is to remove the test since it's not properly testing anything?

Reply:
@EBusch I'd like to replace it with a test that is testing something, if possible

Reply:
<img width="433" alt="image" src="https://github.com/gowithfloat/Float.FileDownloader/assets/1831774/50f9934a-597f-4096-8de7-d8105738c055">
The issue is I can't seem to get the cookie back out of the request (via the response) in code to verify. I was able to see the cookie set via Proxyman.

Reply:
@EBusch You don't want to be checking the response for a cookie--you need to validate that the request does not have a cookie after the server previously set a cookie. The issue [had two suggestions](https://clientportal.atlassian.net/browse/SL-7099?focusedCommentId=518362) for how to handle that.